src/components/AgGrid/TopicEntityTagActions.jsx

```diff
@@ -1,20 +1,44 @@
 import React, {useState} from 'react';
 import {useSelector} from "react-redux";
 import axios from "axios";
+import Modal from "react-bootstrap/Modal";
+import Button from 'react-bootstrap/Button';
 
 export default (props) => {
     const accessToken = useSelector(state => state.isLogged.accessToken);
     const oktaMod = useSelector(state => state.isLogged.oktaMod);
     const testerMod = useSelector(state => state.isLogged.testerMod);
     const accessLevel = (testerMod !== 'No') ? testerMod : oktaMod;
+    const [showModal, setShowModal] = useState(false);
+    const [confirmDelete, setConfirmDelete] = useState(false);
+    const handleClose = () => { setShowModal(false);}
+    const handleConfirm = () => {
+        setConfirmDelete(true);
+        handleDeleteConfirm();
+    }
+    const [title, setTitle] = useState("Please confirm delete here:");
+    const [body, setBody] = useState("body here1");
 
     const handleDeleteClick = async () => {
         let mod = props.data.topic_entity_tag_source.secondary_data_provider_abbreviation;
-        let id = props.data.topic_entity_tag_id;
+
         if (mod !== accessLevel) {
             console.error("Permission denied. Cannot delete this row.");
             return;
         }
+        let topic=props.data.topic_name;
+        let entity_type=props.data.entity_type_name;
+        let entity=props.data.entity_name;
+
+        let noData=props.data.negated;
+        let novelData=props.data.novel_topic_data;
+        setBody("topic:" + topic+" entity type:"+entity_type + " entity:"+ entity + " no Data:"+noData+ " Novel Data:" + novelData);
```

Comment:
I would like to see this data on different lines so that its easier to read rather than all on one line

Reply:
@pjhale good suggestion, fixed now.